android/sdl_android/src/main/java/com/smartdevicelink/managers/SdlManagerListener.java

```diff
@@ -54,7 +54,7 @@
      * @param info info regarding the error
      * @param e    the exception
      */
-    void onError(String info, Exception e);
+    void onError(SdlManager manager, String info, Exception e);
```

Comment:
This is a breaking change, please revert it.

Reply:
@JulianKast Done. (Please, see 20aa80b)